source/Plotter.wl

```diff
@@ -1,25 +1,31 @@
 (*qnms: {{k1, m1, qnms1}, {k2, m2, qnms2}, ...}*)
 
-RewImwPlot[qnms_] := Block[{data},
+Options[RewImwPlot] = Options[ListPlot]
```

Comment:
What? This trick is awesome! I've been trying to do this for quite some time and it's so simple and perfect